### DISCUSSION TITLE
How do i upload an extension

### DISCUSSION THREAD

**Discussion by n-d-v** [TurboWarp/extensions#899] [Q&A] [answered]
I made an extension and i have prepared the .js file and the png file, but when i try to add it is says I have to fork it, and im new to github so I dont know how to use it

**Comment by GarboMuffin** [comment, marked as answer]
yeah thats how github works. you fork -> modify the fork -> make a pull request here -> it will be looked at eventually

**Reply by n-d-v** [reply]
ok thanks!

**Reply by Thebloxers998** [reply]
How should I modify the fork
